frontend/src/component/App.tsx

```diff
@@ -52,76 +51,72 @@ export const App = () => {
 
     return (
         <ErrorBoundary FallbackComponent={LayoutError}>
-            <PlausibleProvider>
-                <ErrorBoundary FallbackComponent={LayoutError}>
-                    <SWRProvider>
-                        <Suspense fallback={<Loader />}>
-                            <ConditionallyRender
-                                condition={!hasFetchedAuth}
-                                show={<Loader />}
-                                elseShow={
-                                    <Demo>
-                                        <>
-                                            <ConditionallyRender
-                                                condition={Boolean(
-                                                    uiConfig?.maintenanceMode,
-                                                )}
-                                                show={<MaintenanceBanner />}
-                                            />
-                                            <LicenseBanner />
-                                            <ExternalBanners />
-                                            <InternalBanners />
-                                            <StyledContainer>
-                                                <ToastRenderer />
-                                                <Routes>
-                                                    {availableRoutes.map(
-                                                        (route) => (
-                                                            <Route
-                                                                key={route.path}
-                                                                path={
-                                                                    route.path
-                                                                }
-                                                                element={
-                                                                    <LayoutPicker
-                                                                        isStandalone={
-                                                                            route.isStandalone ===
-                                                                            true
+            <ErrorBoundary FallbackComponent={LayoutError}>
```

Comment:
duplicate error boundary

Reply:
Interesting, we had it already before. I will remove